git-branchless/tests/command/test_hide.rs

```diff
@@ -168,7 +168,14 @@ fn test_branches_always_visible() -> eyre::Result<()> {
     git.run(&["branch", "test"])?;
     git.run(&["checkout", "master"])?;
 
-    git.run(&["hide", "test", "test^"])?;
+    let (stdout, _stderr) = git.run(&["hide", "test", "test^"])?;
```

Comment:
Let's add a test case which hides more than one branch. Otherwise we don't get to show off all of your string-rendering logic 😉

Reply:
😄 Yeah, that makes sense. I added a test for hiding/deleting 2 branches, and tried to set it up confirm the sorting of the branch names, too.